deepinv/__init__.py

```diff
@@ -55,7 +55,7 @@
 
 __all__ += ["unfolded"]
 
-from deepinv.training_utils import train, test
+from deepinv.training_utils import train, test, train_normalizing_flow

```

Comment:
```suggestion
from deepinv.training_utils import train, test

```

I would remove for now because we are currently doing a big refactoring of the `training_utils.py` file (see my comment over there)